diff -rupN fluidsynth-2.4.7.old/fluidsynth.service.in fluidsynth-2.4.7/fluidsynth.service.in
--- fluidsynth-2.4.7.old/fluidsynth.service.in	2025-07-14 18:13:38.000000000 +0200
+++ fluidsynth-2.4.7/fluidsynth.service.in	2025-07-21 18:38:29.100480949 +0200
@@ -26,7 +26,7 @@ NotifyAccess=main
 Environment=OTHER_OPTS= SOUND_FONT=
 EnvironmentFile=@FLUID_DAEMON_ENV_FILE@
 EnvironmentFile=-%h/.config/fluidsynth
-ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/fluidsynth -is $OTHER_OPTS $SOUND_FONT
+ExecStart=@CMAKE_INSTALL_FULL_BINDIR@/fluidsynth -is $OTHER_OPTS "$SOUND_FONT"
 ExecStartPre=touch /run/lock/fluidsynth/fluidsynth.lock
 ExecStopPost=rm -f /run/lock/fluidsynth/fluidsynth.lock
 
